portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,82 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+/**
+ * Class representing a comment.
+ */
+public class Comment {
+
+  /*The comment left. */
+  private String comment;
+
+  /*The name of who left the comment. */
+  private String commenter;
+ 
+  /*The time and date when the comment was submitted.*/
+  private Date timeSubmitted;
+
+  /** 
+   * Creates a new comment.
+   * @param comment the comment left by the user.
+   * @param commenter the user who left the comment.
+   * @param timeSubmitted the time when a comment was submitted. 
+   */
+  public Comment(String comment, String commenter, Date timeSubmitted){
+    this.comment = comment;
+    this.commenter = commenter;
+    this.timeSubmitted = timeSubmitted;  
+  }
+  
+  /**
+   * Returns the comment left by the user.
```

Comment:
Great job with documenting the code! For some of these methods, you can probably get rid of one of them, i.e. this could be either:

```
/** Returns the comment left by the user. */
```

or:

```
/** @return the comment submitted */
```